source/_locale/fr/LC_MESSAGES/sphinx.po

```diff
@@ -93,10 +93,10 @@ msgstr "Menu"
 msgid "User guide"
 msgstr "Guide de l'utilisateur"
 
-#: ../../source/_templates/header.html:33
-#: ../../source/_templates/header.html:59
-msgid "Tutorials"
-msgstr "Tutoriels"
+#: ../../source/_templates/header.html:32
+#: ../../source/_templates/header.html:46
+msgid "https://academy.opendatasoft.com/page/homepage"
```

Comment:
You cannot modify the translation files directly, they will be overridden when we next get the translation from transifex

Reply:
I see didn't know that… I'll try to find some sort of if/ternary based on locale